packages/now-cli/src/util/output/create-output.ts

```diff
@@ -71,10 +73,21 @@ export default function createOutput({ debug: debugEnabled = false } = {}) {
     print(`${chalk.red(`Error!`)} ${str}\n`);
     const details = slug ? `https://err.sh/now/${slug}` : link;
     if (details) {
-      print(`${action}: ${details}\n`);
+      print(`${chalk.bold(action)}: ${renderLink(details)}\n`);
     }
   }
 
+  function prettyError(err: Error) {
+    let str = err.message;
+    let link: string | null = null;
+    let action: string | undefined = undefined;
+    if (err instanceof LinkableError) {
```

Comment:
Let's avoid using an `instanceof` because we should be able to do this for API Errors too.